spring/src/main/java/com/alibaba/fescar/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -224,34 +236,126 @@ protected Object wrapIfNecessary(Object bean, String beanName, Object cacheKey)
         }
     }
 
-    private MethodDesc makeMethodDesc(GlobalTransactional anno, Method method) {
-        return new MethodDesc(anno, method);
+    /**
+     * is auto proxy TCC bean
+     *
+     * @param bean the bean
+     * @param beanName the bean name
+     * @return boolean boolean
+     */
+    protected boolean isTccAutoProxy(Object bean, String beanName){
```

Comment:
can move to other class（eg：TCCUtils#isTccAutoProxy(Object bean, String beanName，ApplicationContext applicationContext)）